Docker build ends in CMake error

Hi 

I ran the docker build command but ran into the following error:

```
CMake Error at CMakeLists.txt:22 (add_library):
0.379   Cannot find source file:
0.379 
0.379     mav_trajectory_generation/mav_trajectory_generation/src/motion_defines.cpp
```

I saw that the most recent push to this repo (related to the docker build process) failed a test. Do you have plans to fix this?

Thanks

--- a/README.md
+++ b/README.md
@@ -34,9 +34,12 @@ make
 
 Instead of building the code locally, the Dockerfile can be used to containerize the application.
 This is useful if you don't want to install dependencies in your local environment or if you are
-using OSX.
-
-To build the image use
+using OSX.\
+Before building the docker image 
+```
+git submodule update --init --recursive
+```
+then, to build the image use
 
 ```
 docker build . -t gentrajectory

